<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoPlayer</class>
 <widget class="QWidget" name="videoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="vWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QListWidget" name="playlistWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cbox_speed">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>0.5x</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.0x</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5x</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2.0x</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_shot">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/shot.png</normaloff>:/shot.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_prev">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico/image.qrc">
            <normaloff>:/last.png</normaloff>:/last.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_rewind">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico/image.qrc">
            <normaloff>:/down.png</normaloff>:/down.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_pause">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico/image.qrc">
            <normaloff>:/play.png</normaloff>:/play.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_fast">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico/image.qrc">
            <normaloff>:/up (1).png</normaloff>:/up (1).png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_next">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico/image.qrc">
            <normaloff>:/next.png</normaloff>:/next.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_setting">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico/image.qrc">
            <normaloff>:/setting.png</normaloff>:/setting.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="4" column="5">
         <widget class="QToolButton" name="btn_list">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico/image.qrc">
            <normaloff>:/list.png</normaloff>:/list.png</iconset>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QToolButton" name="btn_open">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico/image.qrc">
            <normaloff>:/file.png</normaloff>:/file.png</iconset>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QToolButton" name="btn_qss">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico/image.qrc">
            <normaloff>:/qss.png</normaloff>:/qss.png</iconset>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSlider" name="volumeSlider">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QToolButton" name="toolButton_4">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico/image.qrc">
            <normaloff>:/value.png</normaloff>:/value.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="lb_total">
          <property name="maximumSize">
           <size>
            <width>43</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>all </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>231</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QSlider" name="positionSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lb_start">
          <property name="maximumSize">
           <size>
            <width>43</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>start </string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QToolButton" name="btn_full">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="ico/image.qrc">
            <normaloff>:/full.png</normaloff>:/full.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ico/image.qrc"/>
 </resources>
 <connections/>
</ui>
